packages/yarnpkg-core/sources/Workspace.ts

```diff
@@ -69,6 +69,7 @@ export class Workspace {
         expandDirectories: false,
         onlyDirectories: true,
         onlyFiles: false,
+        ignore: [`node_modules`, `.git`],
```

Comment:
Shouldn't the `.yarn` folder be ignored as well, btw? Otherwise it'll detect the unplugged directory.

Reply:
Yeah, it should. I forgot about that one.